content/sigs/pipeline-authoring/index.adoc

```diff
@@ -78,14 +78,9 @@ overview: >
 
 ---
 
-This special interest group aims to improve and curate the experience
```

Comment:
@x10an14  Any particular reason for removing this?

Reply:
I believe I found the exact same paragraph somewhere else in the document, let me confirm that and get back to you!

Reply:
Yes, under `> Overview` - If you want the information duplicated, we can add it back! (Overview text is still displayed on page)